.github/workflows/pypi-publish.yaml

```diff
@@ -0,0 +1,35 @@
+name: Publish package to PyPi
+
+on:
+  release:
+    types: [published]
+
+jobs:
+
+  push:
+    runs-on: ubuntu-latest
+    if: startsWith(github.ref, 'refs/tags/harmony-plugin')
+    defaults:
+      run:
+        working-directory: ./tutor-contrib-harmony-plugin
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
```

Comment:
This should be `actions/checkout@v4` - otherwise we're just going to get an automated PR asking us to bump it as soon as you merge this :)